### ISSUE TITLE
env_sprite texture browser broken

### ISSUE THREAD

**Issue by SpyceTewan** [TeamSpen210/HammerAddons#61]
- `.vmt` is not appended to the path
- Doesn't automatically filter for `sprite`

**Comment by Luke18033** [comment]
No, the automatic filtering is annoying, and it's not hard to add `.vmt` yourself (or make the poscompiler do it automatically). I actually changed all of these to `material` at one point, I guess TeamSpen changed them back?

**Comment by Luke18033** [comment]
These should all be swapped back to `sprite` now, only the decal ones are changed
